venues/ICLR.cc/2018/Conference/process/withdrawProcess_reveal.js

```diff
@@ -0,0 +1,34 @@
+function() {
+    var or3client = lib.or3client;
+
+    var CONF = 'ICLR.cc/2018/Conference';
+    var BLIND_INVITATION = CONF + '/-/Blind_Submission';
+    var WITHDRAWN_INVITATION = CONF + '/-/Withdrawn_Submission';
+
+    or3client.or3request(or3client.notesUrl + '?id=' + note.referent, {}, 'GET', token)
+    .then(result => {
+        var blindedNote = result.notes[0];
+
+        var milliseconds = (new Date).getTime();
+        blindedNote.ddate = milliseconds
+        return blindedNote;
+    })
+    .then(blindedNote => or3client.or3request(or3client.notesUrl, blindedNote, 'POST', token))
+    .then(result => {
+        console.log('result: ' + JSON.stringify(result));
+        var withdrawn_submission = {
+            original: result.original,
```

Comment:
I like the idea of being attached to the original!

Reply:
we need to fix this queries and get the overwriting notes that are not deleted:

https://github.com/iesl/openreview/blob/master/models/query.js#L296